src/pool-bin/libraries/BinPool.sol

```diff
@@ -80,7 +80,6 @@ library BinPool {
     function initialize(State storage self, uint24 activeId, uint24 protocolFee, uint24 lpFee) internal {
         /// An initialized pool will not have activeId: 0
         if (self.slot0.activeId != 0) revert PoolAlreadyInitialized();
-        if (activeId == 0) revert PoolInvalidParameter();
```

Comment:
can be removed now as `PriceHelper.getPriceFromId(activeId, binStep);` will revert if pool initialized with `activeId == 0`